patches/chrome-android-java-src-org-chromium-chrome-browser-suggestions-tile-SuggestionsTileView.java.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/SuggestionsTileView.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/SuggestionsTileView.java
+index 219aad30b54a4cdbd9530b7f908ec07c4910b174..b0c0fc9c0f65dc188174691b8534f588b47dcf6d 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/SuggestionsTileView.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/SuggestionsTileView.java
+@@ -75,6 +75,7 @@ public class SuggestionsTileView extends TileView {
+             params.height = resources.getDimensionPixelSize(R.dimen.tile_view_icon_size_modern);
+             params.topMargin =
+                     resources.getDimensionPixelSize(R.dimen.tile_view_icon_margin_top_modern);
++            params.width = resources.getDimensionPixelSize(R.dimen.brave_tile_view_icon_size_modern);params.height=resources.getDimensionPixelSize(R.dimen.brave_tile_view_icon_size_modern);params.topMargin=resources.getDimensionPixelSize(R.dimen.zero_margin);
```

Comment:
`super.setIconViewLayoutParams` instead?

Reply:
@bridiver in that case we need to create patch for `SuggestionTileVIew` in layout files.